packages/vite/src/node/optimizer/optimizer.ts

```diff
@@ -166,14 +164,10 @@ async function createDepsOptimizer(
     // Initialize discovered deps with manually added optimizeDeps.include info
 
     const deps: Record<string, string> = {}
-    await addManuallyIncludedOptimizeDeps(deps, config, ssr)
-
-    const discovered = await toDiscoveredDependencies(
-      config,
-      deps,
-      ssr,
-      sessionTimestamp
-    )
+    const [,discovered] = await Promise.all([
+      addManuallyIncludedOptimizeDeps(deps, config, ssr),
+      toDiscoveredDependencies(config, deps, ssr, sessionTimestamp)
+    ])
```

Comment:
`addManuallyIncludedOptimizeDeps` changes `deps` variable and `toDiscoveredDependencies` uses `deps`. So it cannot be parallel.